frontend/office.js

```diff
@@ -119,7 +119,18 @@ $(() => {
       width: "100%",
       height: "100%",
       parentNode: document.querySelector("#meet"),
+      configOverwrite: {
+        //startAudioOnly:true,
+        resolution :360,
+        constraints: {
+          video: {
+            ideal: 360,
+            max:360
+          }
+        }   
+      },
       interfaceConfigOverwrite: {
+        DISABLE_VIDEO_BACKGROUND:true,
```

Comment:
lint it 💅

Reply:
could you update my branch with this lint? :)